lib/screens/custom_widgets/initial_draft_detail_dialog.dart

```diff
@@ -143,28 +154,84 @@ class _DraftWidgetState extends State<_DraftDetailDialog> {
     );
   }
 
+  void navigateToPreviewScreen({required BuildContext context, required NFT nft}) {
+    context.read<EaselProvider>().setPublishedNFTClicked(nft);
+    context.read<EaselProvider>().setPublishedNFTDuration(nft.duration);
+    Navigator.of(context).pushReplacementNamed(RouteUtil.ROUTE_PREVIEW_NFT_FULL_SCREEN);
+  }
+
+  String getImageUrl(CreatorHubViewModel creatorHubViewModel) {
+    if (creatorHubViewModel.nft.assetType == kImageText) {
+      return creatorHubViewModel.nft.url;
+    } else {
+      return creatorHubViewModel.nft.thumbnailUrl;
+    }
+  }
+
   Widget buildRow({required String title, required String subtitle}) {
     return Row(
       children: [
         Expanded(
             child: Padding(
-              padding: EdgeInsets.only(left:isTablet? 20.w : 40.w, right: 5.w),
-              child: Text(
-                title,
-                style: _rowTitleTextStyle,
-              ),
-            )),
+          padding: EdgeInsets.only(left: isTablet ? 20.w : 40.w, right: 5.w),
+          child: Text(
+            title,
+            style: _rowTitleTextStyle,
+          ),
+        )),
         Expanded(
             child: Padding(
-              padding: EdgeInsets.only(right:5.w,),
+                padding: EdgeInsets.only(
+                  right: 5.w,
+                ),
+                child: Row(
+                  children: [
+                    Text(
+                      subtitle.substring(0, 8),
+                      style: _rowTitleTextStyle,
+                    ),
+                    Text(
+                      "...",
+                      style: _rowTitleTextStyle,
+                    ),
+                    Text(
+                      subtitle.substring(subtitle.length - 5, subtitle.length),
```

Comment:
what if the subtitle lenght is 2. This will throw error . Kindly handle this negative case too.